src/api/books/useGetBooksId.ts

```diff
@@ -1,9 +1,14 @@
 import { useState, useEffect } from "react";
 import useApi from "../../hook/useApi";
 import { compareExpect } from "../../util/typeCheck";
+import { Book } from "../../type";
 
-const useGetBooksId = ({ setSelectedBooks, closeModal }) => {
-  const [bookId, setBookId] = useState(null);
+type Props = {
+  setSelectedBooks: React.Dispatch<React.SetStateAction<Book[]>>;
```

Comment:
```suggestion
  setSelectedBooks: React.Dispatch<React.SetStateAction<Book[]>>;
```
혹시 타입을 일반 함수으로 넣어주면 어떨까요?

Reply:
setSelectedBooks(prev => [...prev, newBook]) 식으로 사용하기 때문에 현재 타입을 유지해야 하는군요!